### DISCUSSION TITLE
Can't run anything on python

### DISCUSSION THREAD

**Discussion by hmseanlee** [microsoft/vscode-jupyter#12835] [Questions and Answers]
Failed to start the Kernel. 
Kernel Python 3.10.6 is not usable. Check the Jupyter output tab for more information. 
View Jupyter [log](command:jupyter.viewOutput) for further details.

**Comment by DonJayamanne** [comment]
please reply here

https://github.com/microsoft/vscode-jupyter/issues/12836
